internal/providers/terraform/azure/testdata/monitor_action_group_test/monitor_action_group_test.golden

```diff
@@ -1,37 +1,49 @@
- OVERALL TOTAL                                                                            $145.73 
+ Name                                                                  Monthly Qty  Unit                      Monthly Cost 
+                                                                                                                           
+ azurerm_monitor_action_group.everything_example_with_usage                                                                
+ ├─ Email notifications (2)                                                 26,000  emails                           $0.52 
+ ├─ ITSM connectors (1)                                                     13,000  events                          $65.00 
+ ├─ Push notifications (1)                                                  13,000  notifications                    $0.26 
+ ├─ Secure web hooks (1)                                                    13,000  notifications                    $0.78 
+ ├─ Web hooks (2)                                                           26,000  notifications                    $0.16 
+ ├─ SMS messages                                                                                                           
+ │  └─ Country code 1 (1)                                                   13,000  messages                        $83.85 
+ └─ Voice calls                                                                                                            
+    └─ Country code 86 (1)                                                  13,000  calls                          $780.00 
+                                                                                                                           
+ azurerm_monitor_action_group.everything_example_without_usage                                                             
+ ├─ Email notifications (2)                                     Monthly cost depends on usage: $0.00002 per emails         
+ ├─ ITSM connectors (1)                                         Monthly cost depends on usage: $0.005 per events           
+ ├─ Push notifications (1)                                      Monthly cost depends on usage: $0.00002 per notifications  
+ ├─ Secure web hooks (1)                                        Monthly cost depends on usage: $0.00006 per notifications  
+ ├─ Web hooks (2)                                               Monthly cost depends on usage: $0.000006 per notifications 
+ ├─ SMS messages                                                                                                           
+ │  └─ Country code 1 (1)                                       Monthly cost depends on usage: $0.00645 per messages       
+ └─ Voice calls                                                                                                            
+    └─ Country code 86 (1)                                      Monthly cost depends on usage: $0.06 per calls             
```

Comment:
Doesn't read right when the units are in the plural. Maybe worth introducing the notion of single/multiple in units in the future.